main.go

```diff
@@ -0,0 +1,42 @@
+package main
+
+import (
+	net_http "net/http"
+	"os"
+	"riot-developer-proxy/handlers/http"
+	"riot-developer-proxy/internal/services"
+	"time"
+
+	"github.com/joho/godotenv"
+	"github.com/labstack/echo/v4"
+	"github.com/labstack/echo/v4/middleware"
+)
+
+func init() {
+	if err := godotenv.Load(); err != nil {
```

Comment:
I think I'd rather have README instructions for how to set env vars per platform than add another library. Either that or Makefile, but it might actually be kind of hard to install `make` on Windows so we'd need to include instructions on that too. Or we could have the env vars included as part of the startup command in the README

Reply:
I wouldn't like to setup env vars directly on the host machine.
The other option would be set environment at the docker container, but it would couple the project with docker so not a good option too.
Can we discuss more about it? Why do you think is better to set env vars direct at the platform than to use dotenv?